builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -138,15 +139,25 @@ node() {
     isEventSchdld = true
   }
 
-  def requestId = utilModule.generateRequestId()
-  if (requestId != null) {
-    events.setRequestId(requestId)
-    environmentDeploymentMetadata.setRequestId(requestId)
+  if(request_id == 'none') {
+    requestId = utilModule.generateRequestId()
+    if (requestId != null) {
+        events.setRequestId(requestId)
+        environmentDeploymentMetadata.setRequestId(requestId)
+    } else {
+        error "Request Id Generation failed"
+      }
   } else {
-    error "Request Id Generation failed"
+    echo "Setting up request_id"
+    requestId = request_id
   }
 
-  events.sendStartedEvent('CREATE_DEPLOYMENT', 'Function deployment started', environmentDeploymentMetadata.generateDeploymentMap("started", environment_logical_id, gitCommitHash), environment_logical_id)
+
+  if(request_id == 'none') {
```

Comment:
This can be included in the above condition.

Reply:
done